eva-accession-import-automation/run_accession_import_jobs.py

```diff
@@ -95,8 +101,10 @@ def run_jobs(command_line_args):
     parser = argparse.ArgumentParser(
         description='Run the accession import process for a given species', add_help=False,
         formatter_class=RawTextHelpFormatter)
-    parser.add_argument("-s", "--species", help="Species for which the process has to be run", required=True)
-    parser.add_argument("--scientific-name", help="Scientific name for the species (e.g. sus_scrofa)", required=True)
+    parser.add_argument("-s", "--species",
+                        help="Species for which the process has to be run (e.g. red_sheep_469796)", required=True)
+    parser.add_argument("--scientific-name",
+                        help="Filesystem-friendly scientific name for the species (e.g. sus_scrofa)", required=True)
```

Comment:
Could you align the example with the corresponding scientific name for red sheep?